src/index.js

```diff
@@ -1,87 +1,65 @@
-  }
+// GET
+async function fetchScore() {
+  const response = await fetch(baseURL);
+  return response.json();
 }
 
-// DOM
-const scoreContainer = document.querySelector('.scores');
-const formContainer = document.querySelector('.form');
-
-// Form
-const ul = document.createElement('ul');
-const liOne = document.createElement('li');
-const inputOne = document.createElement('input');
-const inputTwo = document.createElement('input');
-const liTwo = document.createElement('li');
-const button = document.createElement('button');
-button.classList.add('button');
-const btnClass = document.querySelector('.button');
-const submit = document.createTextNode('Submit');
-button.appendChild(submit);
-formContainer.appendChild(ul);
-ul.appendChild(liOne);
-liOne.appendChild(inputOne);
-ul.appendChild(liTwo);
-liTwo.appendChild(inputTwo);
-formContainer.appendChild(button);
-
-const iterateScore = () => {
-  scores.forEach((item, i) => {
-    scores[i].index = i;
-    const div = document.createElement('div');
-    div.classList.add('container');
-    const name = document.createElement('p');
-    const getName = document.createTextNode(item.name);
-    name.appendChild(getName);
-    const number = document.createElement('p');
-    const getScore = document.createTextNode(item.score);
-    number.appendChild(getScore);
-    div.appendChild(name);
-    div.appendChild(number);
-    scoreContainer.appendChild(div);
+// POST
+// My game ID NVMs7bDqCFiWd9Tmg47Y: console.log(response.json())
+// First we create the ID for the object { name: 'Franks Game'}
+async function postScore(objectData) {
+  const response = await fetch(baseURL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(objectData),
   });
-};
+  console.log(response.json());
+}
 
-const clearPrevious = () => {
-  const oldList = document.querySelectorAll('.container');
-  [...oldList].forEach((e) => e.remove());
-};
+// RENDER
+const renderScore = () => {
+  fetchScore()
+    .then((serverData) => {
+      let html = '';
+      serverData.result.forEach((data) => {
+        const htmlSegment = `<p>${data.user}: ${data.score}</p>`;
+        html += htmlSegment;
+      });
 
-const displayScore = () => {
-  scores = getStored();
-  clearPrevious();
-  iterateScore();
+      const container = document.querySelector('.container');
+      container.innerHTML = html;
+    });
 };
 
-const pushScore = () => {
-  const newScore = new AddScore(inputOne.value, inputTwo.value);
-  scores.push(newScore);
-  inputOne.value = '';
-  inputTwo.value = '';
-  setStored();
-  displayScore();
+// SUBMIT
+const getScore = () => {
+  const name = document.querySelector('.name').value;
+  const scores = document.querySelector('.scores').value;
+  postScore({ user: name, score: scores })
+    .then(() => {
+      document.querySelector('.name').value = '';
+      document.querySelector('.scores').value = '';
+      renderScore();
```

Comment:
- [ ] Great job implementing the functions, but running your app using `live server` it does not function as expected. I don't know if it functioned normally at your end? This I cannot not know because you did not even add a README file and maybe with screenshots. So please make sure with your app, one is able to see the scores after submitting and clicking the `Refresh` button. 
![image](https://user-images.githubusercontent.com/62849872/147301903-54763697-16a4-4459-813e-d6d61cb4dd0d.png)
After fill my name and score and submitting, and then click on the refresh button, I cannot find the score been displayed. 
![image](https://user-images.githubusercontent.com/62849872/147301978-95d15457-9187-4b61-aa73-cde49b8b2e33.png)
Please make sure your app is working as expected. 
![image](https://user-images.githubusercontent.com/62849872/147302050-a99a6c22-5931-4117-8423-7c4dc4af0524.png)